libs/deal-ticket/src/deal-ticket.tsx

```diff
@@ -26,9 +19,9 @@ export const DealTicket = ({
   market,
   submit,
   transactionStatus,
-  defaultOrder = DEFAULT_ORDER,
+  defaultOrder,
 }: DealTicketProps) => {
-  const [order, updateOrder] = useOrderState(defaultOrder);
+  const [order, updateOrder] = useOrderState(market, defaultOrder);
```

Comment:
chore (blocking): maybe I am being thick in some way but I am not seeing any validation on position size in here? Even with a step of 0.1 someone could enter 0.001, which we will need to catch and show an error for

Reply:
Probably as form validation 👍